ci/jobs/quick-lint.sh

```diff
@@ -53,6 +53,9 @@ ci/scripts/clang-format.sh $tgt_branch
 echo -e "\n### Check formatting on header guards"
 ci/scripts/include-guard.sh $tgt_branch
 
+echo -e "\n### Check formatting of files on allow list with Verible"
+ci/scripts/verible-format.sh
```

Comment:
```suggestion
ci/scripts/verible-format.sh || true
```

to match the "errors are ignored" behavior in CI